meta-luneos/recipes-support/pidgin/icyque_git.bb

```diff
@@ -0,0 +1,27 @@
+SUMMARY = "WIM Protocol plugin for ICQ for Adium, Pidgin, Miranda and Telepathy IM Framework"
+SECTION = "webos/services"
+LICENSE = "GPLv3.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=1ebbd3e34237af26da5dc08a4e440464"
+
+DEPENDS = "pidgin json-glib"
+
+PV = "0.1.0+gitr${SRCPV}"
```

Comment:
According to https://github.com/EionRobb/icyque/blob/master/libicyque.c#L1734 it should be "0.1+git${SRCPV}"

Reply:
I really just put a random number because I didn't find any releases/tags. But updated now ;)